exercises/I-array-properties/exercise.js

```diff
@@ -6,7 +6,7 @@
 */
 
 function isEmpty(arr) {
-  return; // complete this statement
+  return !arr.length; // complete this statement
```

Comment:
Fantastic use of ! (logical not operator).

Reply:
> Fantastic use of ! (logical not operator). Very clean and easy to read.

Thank you Selchuk